### PR TITLE
update main with the hunt for new bottles flow and functionality

### PR DESCRIPTION
**Pull request by Zclarkwilliams**
updating the code and functions for the main to only consist of the working flow to get new bottles list and update stored inventory of whiskey.

Removed any code relating to price change get and store  due to it not working.

### CODE REVIEW

File: src/lambda_function.py [hunk 1]
```diff
@@ -5,24 +5,31 @@
...
 import logging
-import PyPDF2
-import urllib.request
-from queue import Empty, Full
-#from prettytable import PrettyTable 
-from selenium import webdriver
+from queue import Empty
+from webDriver import WebDriverWrapper
 from selenium.webdriver.common.by import By
-from src.webDriver import WebDriverWrapper
```
Comment (scott-pickthorn) [inline src/lambda_function.py:17]: why is this changed from src.webDriver to webDriver? i guess if we set our python path to src then this will work.

Comment (Zclarkwilliams) [reply src/lambda_function.py:17]: Because that webdriver is in the same file we do not need src., it is unnecessary

**Comment by Zclarkwilliams** [comment]
matched requirements and tested on aws test with success.
